lib/files.js

```diff
@@ -3,10 +3,12 @@ const path = require('path');
 
 module.exports = {
     getCurrentDirectoryBase: () => {
+        // gets the basename of the directory
         return path.basename(process.cwd());
     },
 
     directoryExists: (filePath) => {
+        // gets the whole directory list
         return fs.existsSync(filePath);
     }
 };
```

Comment:
Let's put extra lines at end of every file?

Reply:
yup!